doc/lightning-delpay.7.md

```diff
@@ -0,0 +1,83 @@
+lightning-delpay -- Command for removing a completed or failed payment
+============================================================
+
+SYNOPSIS
+--------
+
+**delpay** *payment_hash* \[status\]
+
+DESCRIPTION
+-----------
+
+The **delpay** RPC command removes a payment as given in **listsendpays** or **listpays** with a complete or failed
+status. However, the command doesn't permit to remove a pending payment.
+
+- *payment_hash*: Rapresents the unique identifier of a payment. To find it, the caller can run **listpays** or **listsendpays**;
+- *status*: Represents a payment status. Otherwise, if it is not specified it is equal to *complete*.
```

Comment:
You should list here the possible values.

```
*status* is the expected status of the payment.
It can be *complete* or *pending*.
Only delete if the payment status matches.
If not specified, defaults to *complete*.
```

Reply:
Thanks for this suggestion. modify made into https://github.com/ElementsProject/lightning/commit/627a374820dd03876eae9e4a99d661cb39f63934